projects/models.py

```diff
@@ -235,16 +238,22 @@ class SectionTypes(models.TextChoices):
         db_index=True,
     )
     override_permissions = models.ManyToManyField(User, through="ProjectUserPermission")
-    # Not discoverable, but can still be accessed via the URL
-    unlisted = BooleanField(default=False, db_index=True)
+
+    visibility = models.CharField(
+        choices=Visibility.choices,
+        default=Visibility.NOT_IN_MAIN_FEED,
+        db_index=True,
+    )
```

Comment:
I'm not sure if I'm a fan of having a default of not in main feed.
Tags, for instance, seem like they should have no visibility setting. Having visibility settable for Tags seems like it would intuivitely do something.
Can we instead have a `Visibility.NOT_APPLICABLE = ""` choice or make this field blank by default?

Also, help text would be good to add while we're here.
Maybe:
```
help_text="""Sets the visibility of this project.
    'Normal' is visible on the main feed, contributes to global leaderboards/medals, and lists the project in the tournaments/question series page. 
    'not_in_main_feed' is not visible in the main feed, but can be searched for, doesn't contribute to global leaderboards/medals, and lists the project in the tournaments/question series page.
    'unlisted' is not visible in the main feed, not searchable, and doesn't contribute to global leaderboards/medals. 

    If this project is not of type "site_main", "tournament", or "question_series", this field should be left blank ''
"""
```

Reply:
Yeah, I understand your point. I don’t want to introduce extra states either. Right now, whether a project should be shown in the main feed is determined by checking all of its projects, including secondary ones, where tags are typically located. Since tags are still considered projects in this model, I think it makes sense for them to always be set to `NOT_IN_MAIN_FEED` to remain neutral. This aligns naturally with the overall visibility paradigm.

Reply:
Added admin construction to force `NOT_IN_MAIN_FEED` for such cases -- https://github.com/Metaculus/metaculus/blob/2c41909d01ad46634c01b5dae5e52471a8014209/projects/admin.py#L249

Reply:
great, no problem.
We can potentially improve (doesn't have to be in this PR) the admin panel to make that field uneditable for Tag instances.
Or, even better, we make Tags / Topics / etc a different entity than Project...